src/debug/debug.cpp

```diff
@@ -1588,7 +1580,11 @@ int32_t DEBUG_Run(int32_t amount,bool quickexit) {
 	else {
 		// ensure all breakpoints are activated
 		CBreakpoint::ActivateBreakpoints();
-		SDL_RaiseWindow(GFX_GetSDLWindow());
+
+		const auto graphics_window = GFX_GetSDLWindow();
+		SDL_RaiseWindow(graphics_window);
+		SDL_SetWindowInputFocus(graphics_window);
```

Comment:
`SDL_RaiseWindow` should already set input focus to the raised window, https://wiki.libsdl.org/SDL_RaiseWindow

Reply:
Thanks @kklobe; will drop that.

Reply:
@kklobe , can you try this:
 3. At the Z:\> prompt: `DEBUG.COM IMGMOUNT.COM`
 4. The debugger window will be raised automatically, no mouse action is needed.
 5. Tap F10 seven times to step instruction by instruction
 6. On seventh tap, the main gfx window is in foregrounded and (should!) be in-focus, press a keyboard key to confirm (again, no mouse action needed).

For me:
 - When the SDL_SetWindowInputFocus calls are commented out, step 6 doesn't focus my keyboard input after the 7th F10 tap.  The main GFX window is in the foreground, but keyboard input appears frozen. 
 - When the SDL_SetWindowInputFocus calls are left in, step 6 (for me) seamlessly transitions from the 7th F10 tap in the debugger window to now having focussed keyboard input in the main gfx window.  I can tap a keyboard key and it shows up on the Z:\> prompt.

Reply:
the above sequence works just fine for me on `main`as well as on your branch - that's the behavior I intended and tested when I devved it. `main` also works on my Windows 10 / VS2022 setup.

Reply:
Thanks @kklobe ; definitely know this was in good working order when your PRs landed :👍; very likely entirely Linux-specific. Will do more testing and also before and after the commit that @grapeli bisected in SDL2 (thank you!)